test/hs/Test/Ganeti/TestCommon.hs

```diff
@@ -435,12 +441,26 @@ genIp6Addr = do
   rawIp <- vectorOf 8 $ choose (0::Integer, 65535)
   return $ intercalate ":" (map (`showHex` "") rawIp)
 
+-- | Helper function to convert an integer in to an IPv6 address in textual
+-- form
+ip6AddressFromNumber :: Integer -> [Char]
+ip6AddressFromNumber ipInt =
+  let chunksOf :: Int -> [a] -> [[a]]
+      chunksOf _ [] = []
+      chunksOf n lst = take n lst : (chunksOf n $ drop n lst)
+      lPadTrim :: Int -> a -> [a] -> [a]
+      lPadTrim n p lst
```

Comment:
Please add a comment what this does/example of how output would look. For tired eyes, it is not obvious.